rosdep/python.yaml

```diff
@@ -9922,6 +9922,13 @@ python3-ujson:
     '*': [python3-ujson]
     '7': null
   ubuntu: [python3-ujson]
+python3-ultralytics:
```

Comment:
Since it looks like this is only available via pip, this key should be `python3-ultralytics-pip`.

Reply:
Did not know that, will change :+1: .